io/eolearn/io/local_io.py

```diff
@@ -294,11 +294,11 @@ def execute(self, eopatch, *, filename=None):
 
         channel_count, height, width = image_array.shape
 
-        src_crs = eopatch.bbox.crs.epsg
+        src_crs = {'init': eopatch.bbox.crs.ogc_string()}
```

Comment:
don't use 'init' here, to get rid of (at least some) warnings

Reply:
Without this it doesn't work. Using epsg code failed unit tests. I suggest that this is resolved in the next release.

Reply:
Huh. Something worked for me, but I am now completely lost. I agree - let's take time to understand the changes going to rasterio 1.2...